data-model/SDP-sample-records/ms-object_sample1.json

```diff
@@ -0,0 +1,48 @@
+{
+  "id": "ark:/21198/z1x07bdf",
+  "type": "Manuscript Object",
+  "shelfmark": "Sinai Syriac 2A",
+  "form": "Codex",
+  "has_binding": true,
+  "binding_note": ["Relatively recent; leather on wood showing signs of repair; metal studs (some missing) on front and back and along edges; remnants of clasps (four total, two for edge and one each at top and bottom)"],
+  "condition_note": "",
+  "references": [
+    {"uri": "https://www.zotero.org/groups/2276183/items/EUCERC5C",
+     "citation_key": "Lewis 1894",
```

Comment:
we may not need the citation_key in the SDP as they could be stored in Zotero and returned for SMDL and display purposes using the Zotero API